### PR TITLE
[CP-1117] Prepare message.component for draft implementation

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1117]

**Description**
Refactoring:
- messages.component - logic
- messages.reducer - logic
- fix issues inside other components

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1117]: https://appnroll.atlassian.net/browse/CP-1117?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/messages/components/delete-message-modals/delete-message-modals.component.tsx [hunk 1]
```diff
@@ -3,88 +3,77 @@
...
+          titleLabel={intl.formatMessage(messages.deleteModalTitle)}
+          info={messages.deleteModalBody}
+          onActionButtonClick={onDelete}
+          onCloseButton={onCloseDeletingModal}
+          cancelButtonLabel={intl.formatMessage(messages.deleteModalCancel)}
+          actionButtonLabel={intl.formatMessage(messages.deleteModalAction)}
+        />
+      )}
+
+      {deleting && !error && (
```
Comment (wojcik92michal) [inline packages/app/src/messages/components/delete-message-modals/delete-message-modals.component.tsx:55]: According to ACC, Loader Modal should not be applied for deleting message (the loading state for deleting message is indicated by adding opacity to an affected message)

File: packages/app/src/__deprecated__/renderer/locales/default/en-US.json [hunk 2]
```diff
@@ -420,6 +419,28 @@
   "module.messages.sendButtonTooltipDescription": "Send message",
   "module.messages.textAreaPlaceholder": "Type your message",
   "module.messages.unreadOnly": "Unread only",
+
+  "module.messages.deletedThread": "1 conversation was deleted.",
```
Comment (wojcik92michal) [inline packages/app/src/__deprecated__/renderer/locales/default/en-US.json:423]: The messages that were previously used and now are not should be removed (e.g. `module.messages.deletingModalTitle` or `module.messages.deleteMessageConfirmationModalBody`).

File: packages/app/src/__deprecated__/renderer/locales/default/en-US.json [hunk 3]
```diff
@@ -420,6 +419,28 @@
...
+  "module.messages.deletedThreads": "{number} conversations were deleted.",
+  "module.messages.deleteThreadModalTitle": "Delete Conversations",
+  "module.messages.deleteThreadModalBody": "Do you really want to delete <b>{num, plural, =-1 {all conversations} one {# conversation} other {# conversations}}?</b>",
+  "module.messages.deleteThreadModalCancel": "Cancel",
+  "module.messages.deleteThreadModalAction": "Delete",
+  "module.messages.deletingThreadModalTitle": "Deleting Conversations",
+  "module.messages.deletingThreadModalSubtitle": "Deleting in progress...",
+  "module.messages.deleteThreadModalErrorTitle": "Deleting Conversations",
```
Comment (wojcik92michal) [inline packages/app/src/__deprecated__/renderer/locales/default/en-US.json:431]: (Not related to this line)

The file `packages/app/src/__deprecated__/main/default-news.json` should not be included in this PR.

File: packages/app/src/__deprecated__/renderer/locales/default/en-US.json (hunk 2 above)
Comment (igorbezsmertnyi) [reply packages/app/src/__deprecated__/renderer/locales/default/en-US.json:410]: https://github.com/mudita/mudita-center/pull/827/commits/4efd4e16515f625821527dcfda069a6477530456

File: packages/app/src/messages/components/delete-message-modals/delete-message-modals.component.tsx (hunk 1 above)
Comment (igorbezsmertnyi) [reply packages/app/src/messages/components/delete-message-modals/delete-message-modals.component.tsx]: https://github.com/mudita/mudita-center/pull/827/commits/65fc2a6f831ca12bbabffd8757fe96501fea9427

File: packages/app/src/__deprecated__/renderer/locales/default/en-US.json (hunk 3 above)
Comment (igorbezsmertnyi) [reply packages/app/src/__deprecated__/renderer/locales/default/en-US.json:418]: https://github.com/mudita/mudita-center/pull/827/commits/f5cce241774a21931a04a54278475bd7e3ffa4ae

**Review by wojcik92michal — DISMISSED** [review]

File: packages/app/src/__deprecated__/renderer/locales/default/en-US.json [hunk 4]
```diff
@@ -383,25 +383,12 @@
   "module.messages.contactTooltipDescription": "Contact details",
   "module.messages.conversationDelete": "1 conversation was deleted.",
   "module.messages.conversations": "Conversations",
-  "module.messages.conversationsDelete": "{number} conversations were deleted.",
   "module.messages.conversationsSelectionsNumber": "{num, plural, =-1 {All Conversations} one {# Conversation} other {# Conversations}} selected",
   "module.messages.deleteButton": "Delete",
-  "module.messages.deleteModalBody": "Do you really want to delete <b>{num, plural, =-1 {all conversations} one {# conversation} other {# conversations}}?</b>",
```
Comment (wercix) [inline packages/app/src/__deprecated__/renderer/locales/default/en-US.json:389]: Not sure if we should delete translations. They can still be in use for users with lower center versions.

Comment (igorbezsmertnyi) [reply packages/app/src/__deprecated__/renderer/locales/default/en-US.json:389]: Yes, it can be. But the lower MC's version has their of set of translations

**Review by wercix — DISMISSED** [review]
